### PR TITLE
Crn 1040 Alumni user profile message

### PR DESCRIPTION
**Pull request by lctrt**

### CODE REVIEW

File: apps/crn-server/test/fixtures/users.fixtures.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ export const getGraphQLUser = (
   ...user,
   flatData: {
     alumniSinceDate: null,
+    alumniLocation: null,
```
Comment (peterstarling) [inline apps/crn-server/test/fixtures/users.fixtures.ts:99]: if there no tests for this in this PR and the only fixture change is the addition of a null `alumniLocation` to the fixture I think we need a test for when it comes back from squidex as not null, no?

**Review by peterstarling — CHANGES_REQUESTED** [review]

**Review by peterstarling — APPROVED** [review]
